pkg/arm/arm.go

```diff
@@ -54,8 +54,10 @@ func (g *simpleGenerator) Generate(ctx context.Context, cs *api.OpenShiftManaged
 		"Startup": func(role api.AgentPoolProfileRole) ([]byte, error) {
 			if role == api.AgentPoolProfileRoleMaster {
 				return util.Template(string(masterStartup), nil, cs, map[string]interface{}{
-					"Role":       role,
-					"TestConfig": g.pluginConfig.TestConfig,
+					"IsRecovery":     len(g.pluginConfig.RecoverEtcdFromBackup) > 0,
+					"BackupBlobName": g.pluginConfig.RecoverEtcdFromBackup,
```

Comment:
One route here is to pass this in as an argument to `Generate()` - usually empty, only populated to do a restore